Sudoku/importSudokuViewController.swift

```diff
@@ -54,15 +61,20 @@ class importSudokuViewController: UIViewController {
                         fatalError()
                     }
                     
-                    cell.importNum.text = String(0)
+                    cell.importNum.text = ""
+                    self.sudokuNum[i] = 0
+                    self.hideIndicator()
                 }
             }
-            let no = UIAlertAction(title: "No", style: .destructive, handler: nil)
+            let no = UIAlertAction(title: "No", style: .destructive) { (action) in
+                self.hideIndicator()
+            }
```

Comment:
```suggestion
            let no = UIAlertAction(title: "No", style: .destructive) { _ in
                self.hideIndicator()
            }
```

사용하지 않으면 `와일드카드` 를 사용해주세요!

Reply:
넵 바로 수정하였습니다!